### DISCUSSION TITLE
How to opt-in to "keyworless search"

### DISCUSSION THREAD

**Discussion by papatistos** [chrisgrieser/shimmering-obsidian#163] [Q&A] [answered]
The release notes mention a new feature called "Keywordless search (Opt-in)". What exactly is it and how do I opt-in?

**Comment by chrisgrieser** [comment, marked as answer]
That was a feature I considered but decided not to implement for various reasons. I accidentally left it in the release notes, but fixed that already. When you update, you should have the correct release notes displayed.
